tests/unit/cellguide_pipeline/test_canonical_marker_genes.py

```diff
@@ -22,6 +22,11 @@
 TEST_SNAPSHOT = "realistic-test-snapshot"
 
 
+def mock_get_asctb_master_sheet():
```

Comment:
Since this is being use by none test file, move it out of this file and into a `mocks.py` or something with a similar name